### DISCUSSION TITLE
Request: Login Rewards (first login of the day)

### DISCUSSION THREAD

**Discussion by HomeLandingsDE** [Zrips/CMI#79] [Feature Requests]
Hey, is it possible to build in a feature that gives a player something (a kit / command ect) when logging in for the first time?

So say:

PlayerA logs in at 10 AM on the server
And do it again 5-6 hours later. But only for the login of 10 AM (so his first of the day) he gets this kit / command

I mean not only new players but especially all players.
I know that one could handle this via the login events but then one would always get the kit / command after logging in.

**Comment by HomeLandingsDE** [comment]
#Push

**Comment by mrfloris** [comment]
To be fair, isn't this just a daily CMI Kit or CMI Playreward that you can setup, with a daily login message reminding them " Don't forget to pick up your /dailyreward " which runs the command.

I don't think this has to become a separate feature.
